### ISSUE TITLE
"Error: This repo already has an 'origin' remote, with value '{repository URL}'" (Preparation 2)

### ISSUE THREAD

**Issue by KMiedema** [STA518-01-SS21/community#9]
Getting this error when using the use_github() function after going through the GitHub first method because the original way was not working for me.

Gives me some code that I would assume is to fix it "usethis::use_git_remote("origin", url = NULL, overwrite = TRUE)" but when I try to type the code in nothing happens and nothing shows up in the GitHub repository I created.

**Comment by dykesb** [comment]
If you are doing the GitHub First method, you would not need to use `usethis::use_git_remote`.  Based on the error message you supplied in the title, it looks like you might have already created a `preparation02` RStudio Project with a psuedo-Git connection ("This repo already has an 'origin' remote...").

I would recommend deleting your current Preparation 2 RStudio Project starting over from Task 1, Step 3 of the [GitHub First method](https://github.com/STA518-01-SS21/community/issues/7#issuecomment-838891521).

**Comment by KMiedema** [comment]
Even after deleting my current Preparation 2 RStudio Project and going through the steps again from the GitHub First method, I still get the same error. I even tried running through it with a different repository name (preparation2 instead of preparation02) and I still end up with the same error. I also tried deleting the repository I created before and starting completely from scratch with the GitHub method and also ended up with the same error.
